apps/nextjs-website/react-components/components/NotFoundPage/NotFoundPage.tsx

```diff
@@ -0,0 +1,146 @@
+'use client';
+import { useEffect } from 'react';
+import {
+  Box,
+  Typography,
+  Link,
+  Stack,
+  useTheme,
+  useMediaQuery,
+} from '@mui/material';
+import { NotFoundPageProps } from '@react-components/types/NotFoundPage/NotFoundPage.types';
+import {
+  IoBackgroundColor,
+  SendBackgroundColor,
+  TextColor,
+} from '@react-components/components/common/Common.helpers';
+
+const safeUseRouter = () => {
+  try {
+    const { useRouter } = require('next/navigation');
+    return useRouter();
+  } catch {
+    return null;
+  }
+};
+
+const localizedTexts = {
+  it: {
+    title: 'Pagina non trovata',
+    body: 'Oops! La pagina che stai cercando non è disponibile.',
+    redirectIntroText: 'Se il reindirizzamento non avviene,',
+    redirectText: 'clicca qui per tornare alla home',
+  },
+  en: {
+    title: 'Page not found',
+    body: 'Oops! The page you are looking for is not available.',
+    redirectIntroText: 'If the redirect does not happen,',
+    redirectText: 'click here to go back to the homepage',
+  },
+  fr: {
+    title: 'Page non trouvée',
+    body: 'Oops ! La page que vous recherchez est introuvable.',
+    redirectIntroText: 'Si la redirection ne fonctionne pas,',
+    redirectText: 'cliquez ici pour revenir à la page d’accueil',
+  },
+  de: {
+    title: 'Seite nicht gefunden',
+    body: 'Die Seite, die Sie suchen, ist nicht verfügbar.',
+    redirectIntroText: 'Wenn die Weiterleitung nicht erfolgt,',
+    redirectText: 'klicken Sie hier, um zur Startseite zurückzukehren',
+  },
+  sl: {
+    title: 'Stran ni najdena',
+    body: 'Stran, ki jo iščete, ni na voljo.',
+    redirectIntroText: 'Če preusmeritev ne uspe,',
+    redirectText: 'kliknite tukaj za vrnitev na domačo stran',
+  },
+};
+
+const NotFoundPage = ({
+  image,
+  mobileImage,
+  redirectUrl = '/',
+  disableRedirect = false,
+  theme,
+  themeVariant,
+  sectionID,
+  locale = 'it',
+}: NotFoundPageProps) => {
+  const muiTheme = useTheme();
+  const isMobile = useMediaQuery(muiTheme.breakpoints.down('sm'));
+  const router = !disableRedirect ? safeUseRouter() : null;
```

Comment:
```suggestion
  const router = useRouter();
```
Follows previous suggestion to remove `safeUseRouter`